tests/telemetry/test_telemetry.py

```diff
@@ -221,8 +221,7 @@ def process_trace(self, trace):
     ]["message"]
 
 
-@pytest.mark.skip(reason="We don't have a way to capture unhandled errors in bootstrap before telemetry is loaded")
```

Comment:
So this is now false because we moved the import right? https://github.com/DataDog/dd-trace-py/blob/5db374c1a63aab3be408b53568e4297461a03f86/ddtrace/__init__.py#L20

Reply:
Yup. Before we only enabled telemetry on ddtrace-run (or when the first event was queued)